src/openloco/windows/vehicle.cpp

```diff
@@ -0,0 +1,174 @@
+#include "../companymgr.h"
+#include "../input.h"
+#include "../interop/interop.hpp"
+#include "../openloco.h"
+#include "../things/thingmgr.h"
+#include "../windowmgr.h"
+
+using namespace openloco::interop;
+
+namespace openloco::ui::vehicle
+{
+    namespace p1
```

Comment:
Do you have any guess as to what p1, p2, and p5 might be? (#136 names a few more things, if it helps.)

Reply:
Those are just the different tabs